pkg/manifest/messages.go

```diff
@@ -0,0 +1,75 @@
+package manifest
```

Comment:
As with the name of the file itself, the package name here is too generic and doesn't say anything about the usage. If this is really focused just on tickler messages only, maybe the package name should be ticklermessages or just `ticklers` ? If the package has grander ambitions, then we should name it based on that.  (vscode should be able to rename this without trouble)

Reply:
See answer above.